Mathlib/Tactic/Linter/TextBased.lean

```diff
@@ -213,8 +219,11 @@ def formatErrors (errors : Array ErrorContext) (style : ErrorFormat) : IO Unit :
     IO.println (outputMessage e style)
 
 /-- Core logic of a text based linter: given a collection of lines,
-return an array of all style errors with line numbers. -/
-abbrev TextbasedLinter := Array String → Array (StyleError × ℕ)
+return an array of all style errors with line numbers and, optionally,
+the collection of lines changed to fix the linter errors automatically.
+(Such automatic fixes are only possible for some kinds of `StyleError`s.)
```

Comment:
This doc-string is still ambiguous about which lines are returned.

Reply:
Indeed... clarified the docstring further.